packages/ui/.storybook/preview.js

```diff
@@ -1,15 +1,26 @@
-import '../styles/index.css';
-import resolveConfig from 'tailwindcss/resolveConfig';
-import tailwindConfig from '@sovryn/tailwindcss-config/index';
+import "../styles/index.css";
+import resolveConfig from "tailwindcss/resolveConfig";
+import tailwindConfig from "@sovryn/tailwindcss-config/index";
+import { MemoryRouter } from "react-router-dom";
 
 const config = resolveConfig(tailwindConfig);
 
 const screens = {
-  xs: '320px',
+  xs: "320px",
```

Comment:
This formatting is not correct, when I format this file I get single quotes that are correct.